src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.vb

```diff
@@ -1262,6 +1251,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             Return method
         End Function
 
+        Private Shared Function GroupByName(symbols As ArrayBuilder(Of PENamedTypeSymbol)) As Dictionary(Of String, ImmutableArray(Of PENamedTypeSymbol))
```

Comment:
>GroupByName [](http://example.com/codeflow?start=32&length=11)

Is there a specific reason for placing this helper here? Overwise, consider placing it right after ```CreateNestedTypes``` that used to do this work.